sdk/sample/kfserving_sdk_sample.ipynb

```diff
@@ -0,0 +1,175 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for KFServing SDK "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is a sample for KFServing SDK. \n",
+    "\n",
+    "The notebook shows how to use KFServing SDK to deploy, get, patch and delete KFService."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "\n",
+    "from kfserving.models.v1alpha1_model_spec import V1alpha1ModelSpec\n",
+    "from kfserving.models.v1alpha1_tensorflow_spec import V1alpha1TensorflowSpec\n",
+    "from kfserving.models.v1alpha1_kf_service_spec import V1alpha1KFServiceSpec\n",
+    "from kfserving.models.v1alpha1_kf_service import V1alpha1KFService\n",
+    "from kfserving.api.kf_serving_api import KFServingApi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define KFService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Firstly define the model spec for default model spec, and then define the kfservice basic on the model spec."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "default_model_spec = V1alpha1ModelSpec(tensorflow=V1alpha1TensorflowSpec(\n",
+    "    model_uri='gs://kfserving-samples/models/tensorflow/flowers'))\n",
+    "\n",
+    "kfsvc = V1alpha1KFService(api_version='serving.kubeflow.org/v1alpha1',\n",
+    "                              kind='KFService',\n",
+    "                              metadata=client.V1ObjectMeta(name='flower-sample'),\n",
+    "                              spec=V1alpha1KFServiceSpec(default=default_model_spec))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create KFService"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Call KFServingAPi to deploy KFService."
```

Comment:
Minor: To make it consistent  s/KFServingAPi/KFServingApi

Reply:
Sorry... I don't quite catch your meaning :-(

Reply:
Oh, got that, one typo :-)... Thanks for the reminder, @johnugeorge